ansible/roles/nbp-installer/tasks/main.yml

```diff
@@ -20,20 +20,24 @@
     mode: 0755
   with_items:
     - "{{ nbp_work_dir }}"
-  when: nbp_plugin_type != "hotpot_only"
+  when: deploy_project != "multicloud" and nbp_plugin_type != "hotpot_only"
```

Comment:
There is no need to change this file because nbp_plugin_type has already indicate if nbp is required.